### PR TITLE
chore: solving some of the docstring issues

### PR DESCRIPTION
**Pull request by uriii3**
Trying to make some progress on [CMT-27](https://cms-change.atlassian.net/browse/CMT-27) regarding the docstrings and the explanations.

### CODE REVIEW

**Comment by uriii3** [comment]
Until PR 86 is not merged it won't pass the tests!

File: copernicusmarine/python_interface/get.py [hunk 1]
```diff
@@ -65,9 +63,11 @@ def get(
         service (str, optional): Force the use of a specific service.
         overwrite_metadata_cache (bool, optional): If True, overwrite the metadata cache.
         no_metadata_cache (bool, optional): If True, do not use the metadata cache.
+        no_directories (bool, optional): If True, downloaded files will not be organized into directories.
+        show_outputnames (bool, optional): If True, display the names of the downloaded files.
         filter (str, optional): Apply a filter to the downloaded data.
         regex (str, optional): Apply a regular expression filter to the downloaded data.
-        file_list (Union[pathlib.Path, str], optional): A path to a text file that list filenames line by line. Filenames must match the absolute paths of the files to download.
+        file_list (Union[pathlib.Path, str], optional): Path to a .txt file containing a list of file paths, line by line, that will be downloaded directly. These files must be from the specified dataset using the --dataset-id. If any files cannot be found, the Toolbox will list all files on the remote server and attempt to find a match.
```
Comment (renaudjester) [inline copernicusmarine/python_interface/get.py:70]:
```suggestion
        file_list (Union[pathlib.Path, str], optional): Path to a .txt file containing a list of file paths, line by line, that will be downloaded directly. These files must be from the specified dataset using the --dataset-id. If no files can be found, the Toolbox will list all files on the remote server and attempt to find a match.
```

File: copernicusmarine/python_interface/get.py
Comment (renaudjester) [inline copernicusmarine/python_interface/get.py:1]: Would be nice if we find a way to centralise the python interface doc strings and the CLI ones

File: copernicusmarine/python_interface/get.py (hunk 1 above)
Comment (uriii3) [reply copernicusmarine/python_interface/get.py]: updated! Thanks for the review :)

**Review by renaudjester — APPROVED** [review]

File: copernicusmarine/python_interface/get.py
Comment (uriii3) [reply copernicusmarine/python_interface/get.py:1]: I opened a ticket in [CMT-72](https://cms-change.atlassian.net/browse/CMT-72) to have it in mind.

**Pull request merged**